GSoC-interest.md

```diff
@@ -19,4 +19,4 @@ GSoC Mentors
 | Name | Email | Idea | Repo | Project Proposal |
 | --- | --- | --- | --- | --- |
 | Georg Link | glink at unomaha.edu | Prototype New CHAOSS Metrics | [GeorgLink/governance](https://github.com/GeorgLink/governance) | [none](https://wiki.linuxfoundation.org/chaoss/gsoc-ideas) |
-| Pranjal Aswani | aswani.pranjal at gmail.com | Reporting of CHAOSS Metrics | [aswanipranjal/chaoss-microtasks](https://github.com/aswanipranjal/chaoss-microtasks) | To-Do |
+| Jacek Dąbrowski | j.dabrowski@cs.ucl.ac.uk | Improve Manuscripts and provide with new CHAOSS Metrics | [jsdabrowski/CHAOSS-project](https://github.com/jsdabrowski/CHAOSS-project) | [TBD]() |
```

Comment:
Hi @jsdabrowski,

I have a few change requests before accepting your pull request.
 - Please update your pull request to not delete other interested students that are already in the list.
 - Please change the format of your email to use "at" instead of @
 - Please change the last column to read "To-Do" without a link, if you don't have your Project Proposal ready yet.